### ISSUE TITLE
[BUG] Landing sometimes break ship.

### ISSUE THREAD

**Issue by Dennok** [voidcrew/Voidcrew-LRP#149]
## Reproduction:

 Land on planet (or another POI) with destroyed pill on it. You need exaxtly land on pill.
 See like half of ship lost in space and half landed on planet.

Seems like turf with crioconsole can't be deleted by shuttle landing. 

![scrnshot1](https://user-images.githubusercontent.com/7734424/153727667-2eb210b7-5479-437d-8177-7b4a6d5644cf.png)

**Comment by Dennok** [comment]
Fortunately you can fly again and return ship to normal state. Except atmos in rooms on break line.

**Comment by Dennok** [comment]
Now i can 100% reproduse it by landing ship on xeno queen.

**Comment by SuperNovaa41** [comment]
Yeah, theres a number of different reasons that thi shappens. I believe they're all caused by ships landing on something and force del'ing it

**Comment by Dennok** [comment]
Runtimes on del, not just force.

**Comment by SuperNovaa41** [comment]
Right yeah, i meant bad dels not force dels
